Detectors/ITSMFT/ITS/workflow/src/ClusterWriterSpec.cxx

```diff
@@ -60,13 +58,10 @@ void ClusterWriter::run(ProcessingContext& pc)
   LOG(INFO) << "ITSClusterWriter pulled " << clusters.size() << " clusters, in "
             << rofs.size() << " RO frames";
 
-  // This conversion vector -> map is just for the convenience of "offline" analyses
-  std::map<int, o2::itsmft::ClusterPattern> map;
-  o2::itsmft::ClusterTopology::makeRareTopologyMap(patterns, map);
-
   TTree tree("o2sim", "Tree with ITS clusters");
```

Comment:
I think this tree creation can be moved to init(), to allow writing separate entry for each TF in case of multiple TFs, perhaps in a different PR.

Reply:
@shahor02 OK. Just moving the tree creation to init() could be done within this PR, I think.